plugin.json

```diff
@@ -58,6 +58,13 @@
                     }
                 ]
             },
+            {
+                "key": "JitsiCompatibilityMode",
+                "display_name": "Enable compatibility mode",
+                "type": "bool",
+                "help_text": "(Insecure) This improve the compatibility with your jitsi instance using jitsi javascript code through a proxy build in the plugin. **WARNING:** This can compromise the security of your mattermost instance if the security of your Jitsi instance get compromised.",
```

Comment:
```suggestion
                "help_text": "(Insecure) Include the JavaScript API hosted on your Jitsi instance directly in Mattermost instead of the default version provided by the plugin. **WARNING:** This can compromise the security of your Mattermost instance if your Jitsi instance is not fully trusted.",
```

Reply:
@jupenur Talked to Jesús about the purpose of this setting. I'm still hesitant to offer an "insecure" option for an admin but the alternatives aren't ideal either, so we can use this as a starting point.

I'm thinking we should adjust the setting to clarify when to use this setting. What do you think for this proposal?

```
Enable Compatibility Mode

(Insecure) If your Jitsi server is not compatible with this plugin, include the JavaScript API hosted on your Jitsi server directly in Mattermost instead of the default API version provided by the plugin. **WARNING:** Enabling this setting can compromise the security of your Mattermost system, if your Jitsi server is not fully trusted and allows direct modification of program files. Use with caution.
```

Feedback welcome!

Reply:
LGTM

Reply:
Changed using your text @jasonblais